terms/structuralriskminimization_maf.tex

```diff
@@ -0,0 +1,99 @@
+\chapter{\textit{}Structural Risk Minimization}
```

Comment:
WHy are you italicing the title ???  Let the chapter def decide how it should format the titles.  Remove.